.github/SUPPORT.md

```diff
@@ -0,0 +1,22 @@
+<!-- Added as a seperate file such that it is highlighted for new contributors: 
+https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project
+-->
+# Help!: I have a question related to Kirby
+
+We use Github discussions for providing support, answering questions, general discussion and announcements; it can be accessed [here](https://github.com/kirbydesign/designsystem/discussions).
+
+If you have a question you are seeking an answer to - you can create a new discussion with your question and set the category to Q&A as this marks it as "unanswered".
+
+We have **Kirby Open Office** every Tuesday and Thursday between 9:15 to 10:15. This is a dedicated timeslot where we answer questions posted to Github discussions. 
+
+**Do not send direct messages to members of team-kirby:** This is not because we do not want to talk to you - we however prefer questions to be asked in public, such that everyone can benefit from the answers. 
+
+# Team Kirby 
```

Comment:
Should Alexander & Marvin be listed here as well? 
Omitted to do it in the first place as I just listed our little cluster of desks of people working full time. 

But considering how big a part of Kirby they are, I guess they should be listed here as well 😄?